frontend/src/pages/NeuroScan.jsx

```diff
@@ -12,31 +14,43 @@ const useStyles = makeStyles((theme) => ({
       },
     },
   },
+  layoutContainer: {
+    position: 'relative',
+    width: '100%',
+    height: '90vh',
+  },
 }));
 
 export default function NeuroScan() {
   const classes = useStyles();
-
+  const store = useStore();
+  const [LayoutManager, setLayoutManager] = useState(undefined);
   const [shrinkSidebar, setShrinkSidebar] = React.useState(false);
 
   const handleToggle = () => {
     setShrinkSidebar(!shrinkSidebar);
   };
 
+  useEffect(() => {
+    // Workaround because getLayoutManagerInstance
+    // is undefined when calling it in global scope
+    // Need to wait until store is ready ...
+    // TODO: find better way to retrieve the LayoutManager component!
+    if (LayoutManager === undefined) {
+      const myManager = getLayoutManagerInstance();
+      if (myManager) {
+        setLayoutManager(myManager.getComponent());
+      }
+    }
+  }, [store]);
+
   return (
     <Box className={classes.root}>
       <Header shrink={shrinkSidebar} toggleSidebar={handleToggle} view={VIEWS?.neuroScan} />
       <Box className="primary-structure" display="flex">
         <LeftSidebar shrink={shrinkSidebar} />
-        <Box className="primary-structure_content">
-          <Box className="MuiBox-empty" display="flex" flexDirection="column" alignItems="center" justifyContent="center">
-            <Typography variant="h2">
-              No Elements Added yet.
-            </Typography>
-            <Typography variant="h2">
-              You can add one with the Search Component.
-            </Typography>
-          </Box>
+        <Box className={`primary-structure_content ${classes.layoutContainer}`}>
+          {LayoutManager === undefined ? <CircularProgress /> : <LayoutManager />}
```

Comment:
Here we would either show the placeholder if no viewers are listed or show the LayoutManager component (aka FlexLayout wrapper)